pkg/api/config.go

```diff
@@ -223,11 +223,18 @@ func validateReleaseTagConfiguration(fieldRoot string, input ReleaseTagConfigura
 	var validationErrors []error
 
 	if len(input.Namespace) == 0 {
-		validationErrors = append(validationErrors, fmt.Errorf("%s: no namespace defined", fieldRoot))
+		validationErrors = append(validationErrors, fmt.Errorf("%s.namespace: must be set", fieldRoot))
 	}
 
 	if len(input.Name) == 0 {
-		validationErrors = append(validationErrors, fmt.Errorf("%s: no name defined", fieldRoot))
+		validationErrors = append(validationErrors, fmt.Errorf("%s.name: must be set", fieldRoot))
+	} else {
+		ok, err := regexp.MatchString(`4\.[0-9]+`, input.Name)
+		if err != nil {
+			validationErrors = append(validationErrors, fmt.Errorf("%s.name: must be of the form 4.x, failed to parse %q: %w", fieldRoot, input.Name, err))
```

Comment:
The `regexp` documentation is not very explicit, but I think `err` here is restricted to regexp compilation.